tests/v2/validation/fleet/provisioning/new_cluster_existing_gitrepo_test.go

```diff
@@ -122,16 +121,10 @@ func (f *FleetWithProvisioningTestSuite) SetupSuite() {
 	f.standardUserClient = standardUserClient
 }
 
-func (f *FleetWithProvisioningTestSuite) TestProvisionAfterAddedGitRepo() {
+func (f *FleetWithProvisioningTestSuite) TestHardenedAfterAddedGitRepo() {
```

Comment:
was just noticing this. qq here. are we using hardened cluster in the tests?

Reply:
This was an update to provision hardened cluster instead of non-hardened. Since hardened is a subset of normal provisioning, it makes the most sense to just change this to hardened vs. having 2 tests for it.